samples/Microsoft.Bot.Samples.Ai.SmartTalk/Microsoft.Bot.Samples.Ai.SmartTalk.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
netstandard2.0

Reply:
See the project file for the Microsoft.Bot.Builder.Core and copy everything from there.